### ISSUE TITLE
False positive on RUN scripts with square brackets

### ISSUE THREAD

**Issue by ColinHebert** [RedCoolBeans/dockerlint#33]
### Issue details

False positive on `RUN` with square brackets `[` in the script.
If a script in `RUN` has square brackets (see [test and \[](https://linux.die.net/man/1/test)) and the expression evaluated contains single quotes, then docker lint will show `Arguments to RUN in exec form must not contain single quotes on line` even though this is not the exec form

### Steps to reproduce/test case

```dockerfile
RUN if [ 'test' = 'test' ]; then echo oops; fi
```

### Affected version(s)

0.3.1
